DoSomethingArchive/legacy-website
# User profile powered by Rogue

#### What's this PR do?
- Updates user profile page to be powered by Rogue when Rogue is on. 

#### How should this be reviewed?
- Go to user profile page.
- You should see all of the campaigns you're doing and have done with data from Rogue.
- Make sure reactions work. **You should be able to add a reaction to the user profile and it will show up in Rogue. However, when you "unlike" something, it doesn't seem to send to Rogue correctly and I think this is a larger bug (tried in the gallery on the campaign page and saw same behavior). 

#### Relevant tickets
Fixes https://www.pivotaltracker.com/n/projects/2019429/stories/149207077

#### Checklist
